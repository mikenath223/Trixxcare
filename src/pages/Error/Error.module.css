.image {
  margin-top: 20px;
  max-width: 68%;
  height: auto;
}

.containerErr {
  text-align: center;
  position: relative;
  width: 98vw;
  height: 90vh;
}

.h1 {
  display: none;
}

.homeBut {
  display: block;
  width: 30%;
  margin: auto;
  text-align: center;
}

.button {
  position: absolute;
  width: 250px;
  bottom: 50%;
  left: calc(50% - (250px/2));
  border: 2px solid silver;
  font-size: 25px;
  font-weight: 900;
  height: 50px;
  border-radius: 20px;
}

.button:focus,
.button:hover {
  border: none;
  outline: none;
}

.homeLink {
  font-weight: 800;
  font-size: 40px;
  padding-top: 10px;
  padding-left: 40px;
  color: #d45436;
}

.navbar {
  padding: 10px 0;
  box-shadow: 0 2px 2px silver;
}
