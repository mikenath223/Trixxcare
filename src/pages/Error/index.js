import React from 'react';
import { Link } from 'react-router-dom';
import styles from './Error.module.css';

const Error = () => (
  <div>
    <nav className={styles.navbar}>
      <Link to="/">
        <h2 className={styles.homeLink}>
          trixxcare
        </h2>
      </Link>
    </nav>
    <div className={styles.containers}>
      <img className={styles.image} src="https://blog.thomasnet.com/hs-fs/hubfs/shutterstock_774749455.jpg?width=900&name=shutterstock_774749455.jpg" alt=".." />
      <h1 data-testid="check-sor-route" className={styles.h1}>404 sor</h1>
      <Link to="/" className={styles.homeBut}><button className={styles.button} type="button">Go Home</button></Link>
    </div>
  </div>
);

export default Error;
