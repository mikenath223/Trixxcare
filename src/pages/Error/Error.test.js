import React from 'react';
import withReduxRouter from 'utils/testUtils';
import Error from './index';


describe('404 page', function () {

  it('should match snapshot', () => {
    const { container } = withReduxRouter(<Error />);

    expect(container).toMatchSnapshot();
  });
});
