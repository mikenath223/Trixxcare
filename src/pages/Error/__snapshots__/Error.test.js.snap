// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page should match snapshot 1`] = `
<div>
  <div>
    <nav
      class="navbar"
    >
      <h2
        data-testid="check-error-route"
      >
        404 Error
      </h2>
      <a
        href="/"
      >
        <h2
          class="homeLink"
        >
          trixxcare
        </h2>
      </a>
    </nav>
    <div
      class="containers"
    >
      <img
        alt=".."
        class="image"
        src="https://blog.thomasnet.com/hs-fs/hubfs/shutterstock_774749455.jpg?width=900&name=shutterstock_774749455.jpg"
      />
      <h1
        class="h1"
        data-testid="check-sor-route"
      >
        404 sor
      </h1>
      <a
        class="homeBut"
        href="/"
      >
        <button
          class="button"
          type="button"
        >
          Go Home
        </button>
      </a>
    </div>
  </div>
</div>
`;
